book/website/ethical-research/data-hazards/dh-case-study.md

```diff
@@ -0,0 +1,281 @@
+(er-datahazardscasestudy)=
+# Case Study: Data Ethics and Reproducibility Symposium and Data Hazards Workshop 
+We (the organisers: Ceilidh Welsh and Susana Roman Garcia) wanted to create an event that revolved around the Data Hazards framework. The event focused on showcasing how different people think about embedding ethics into the work that they do. 
+As PhD students, we both felt the need to connect our interest in creating more ethical and reproducible work with our project's scientific questions. 
+This passion for working more ethically brought us together, and we sought to see how other people are embedding ethics into their work. 
+As part of the Symposium, we hosted talks from PhD students who have been striving to make their work reproducible, how university lecturers might think of implementing Data Hazards into their material, and more (this is available in this {ref}`er-datahazardscasestudy-postevent-githubrepo`). 
```

Comment:
Does "the Symposium" need a little bit more context here? (I know it comes up below, but maybe move some of that context up here..?)